### DISCUSSION TITLE
How do we use existing lambda function with a new route on API?

### DISCUSSION THREAD

**Discussion by newtechfellas** [serverless/serverless#10008] [Q&A]
If aws account already has lambda functions and I want to add http event to them, how do we mention in sls configuration to look for existing lambda function? configuration is expecting handler, name properties and looking for local definition of the function. How do I specify arn of existing lambda function for a new http event trigger?

In short, I want to create new API route but use existing lambda functions (created outside serverless scope) to proxy.

**Comment by pgrzesik** [comment]
Hello @newtechfellas - Framework currently does not offer any specific syntax to address such use case - you would have to configure it via raw CloudFormation - you can find more details about it here: https://www.serverless.com/framework/docs/providers/aws/guide/resources
